root/static/scripts/edit/URLCleanup.js

```diff
@@ -232,6 +226,35 @@ const LINK_TYPES = {
   }
 };
 
+const RESTRICTED_LINK_TYPES = _.reduce([
```

Comment:
It does what it looks like to do. I picked the term _restricted_ from the doc [Style / Relationships / URLs / Restricted relationships](https://musicbrainz.org/doc/Style/Relationships/URLs#Restricted_relationships). Shall I replace it by _whitelisted_ instead?